web/static/js/components/activity/ActivityDescription.jsx

```diff
@@ -69,11 +69,10 @@ class ActivityDescription extends Component {
           case 'start':
             return t('Started <i>{{surveyName}}</i> survey', {surveyName: surveyName})
           case 'request_cancel':
-            return t('Cancelling <i>{{surveyName}}</i> survey', {surveyName: surveyName})
-          case 'completed_cancel':
-            return t('Cancelled <i>{{surveyName}}</i> survey', {surveyName: surveyName})
           case 'stop':
```

Comment:
The `stop` event isn't being registered since [this change](https://github.com/instedd/surveda/commit/3ffaf2e882ffcd960727ef7e6f69bd4569df5c4a#diff-0041c7bf7c8e830ca79758e300963067L197). So, this line remains for backward compatibility.